src/GitHub.VisualStudio/UI/Views/GitHubConnectContent.xaml.cs

```diff
@@ -101,7 +101,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 
             var item = values[0] as ISimpleRepositoryModel;
             var context = values[1] as IGitAwareItem;
-            return context?.ActiveRepoUri == item?.CloneUrl && context?.ActiveRepo?.LocalPath == item?.LocalPath;
+            return context?.ActiveRepoUri == item?.CloneUrl && String.Equals(context?.ActiveRepo?.LocalPath, item?.LocalPath, StringComparison.OrdinalIgnoreCase);
```

Comment:
The implementation uses [the Ordinal comparison as well](https://github.com/github/VisualStudio/blob/master/src/GitHub.Exports/Primitives/StringEquivalent.cs#L32), so == should actually be doing the correct thing here, they're both `UriString`.